sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample1_RecognizeContentFromFile.cs

```diff
@@ -21,49 +21,54 @@ public async Task RecognizeContentFromFile()
 
             FormRecognizerClient client = new FormRecognizerClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
 
-            string invoiceFilePath = FormRecognizerTestEnvironment.CreatePath("Invoice_1.pdf");
+            string filePath = FormRecognizerTestEnvironment.CreatePath("Invoice_1.pdf");
 
-            using (FileStream stream = new FileStream(invoiceFilePath, FileMode.Open))
+            #region Snippet:FormRecognizerRecognizeFormContentFromFile
+            //@@ string filePath = "filePath";
```

Comment:
Just for consistency: in other samples we're using
```
//@@ string modelId = "<modelId>";
//@@ string formFilePath = "<formFilePath>";
```
so maybe we could use `"<filePath>"` here as well.